recipes/pal2nal/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pal2nal" %}
+{% set version = "14" %}
+{% set sha256 = "0e458d38571ab6d61078047a9e80d843bb2e733414eeff8b1144d1c6bc13848f" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: http://www.bork.embl.de/pal2nal/distribution/pal2nal.v14.tar.gz
+
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: True
+
+requirements:
+  host:
+    - perl
+
+  run:
+    - perl
+    - perl-getopt-long
+
+test:
+  commands:
+    - pal2nal.pl -h
+
+
+about:
+  home: http://www.bork.embl.de/pal2nal/
+  license: GPL
+  license_family: GPL
+  license_file: README
+  summary: robust conversion of protein sequence alignments into the corresponding codon alignments
+
+extra:
+  identifiers:
+    - biotools:{{ name }}

```

Comment:
You can actually just use the name here, instead of the jinja thing. It's not like that's ever going to change.

Reply:
got that, Thanks!